### PR TITLE
Add documentation for ActionText::Attachment [ci-skip]

### PR DESCRIPTION
**Pull request by p8**
### Checklist

Before submitting the PR make sure the following are checked:

* [x] This Pull Request is related to one change. Changes that are unrelated should be opened in separate PRs.
* [x] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[Fix #issue-number]`
* [x] Tests are added or updated if you fix a bug or add a feature.
* [x] CHANGELOG files are updated for the changed libraries if there is a behavior change or additional feature. Minor bug fixes and documentation changes should not be included.

### CODE REVIEW

File: actiontext/lib/action_text/attachment.rb [hunk 1]
```diff
@@ -3,6 +3,17 @@
...
+  #
+  # Attachments serialize attachables to HTML or plain text.
+  #
+  #   class Person < ApplicationRecord
+  #     include ActionText::Attachable
+  #   end
+  #
+  #   attachable = Person.create! name: "Javan"
+  #   attachment = ActionText::Attachment.from_attachable(attachable)
+  #   attachment.to_html # => "<action-text-attachment sgid=\"BAh7CEk...
```
Comment (adrianna-chang-shopify) [inline actiontext/lib/action_text/attachment.rb:16]: Might make sense to just add the closing quote on this example? (Same below)
```suggestion
  #   attachment.to_html # => "<action-text-attachment sgid=\"BAh7CEk..."
```

**Review by adrianna-chang-shopify — APPROVED** [review]
Looks good to me!

File: actiontext/lib/action_text/attachment.rb (hunk 1 above)
Comment (p8) [reply actiontext/lib/action_text/attachment.rb]: Thanks @adrianna-chang-shopify ! I think that makes sense.

**Comment by p8** [comment]
Thanks @adrianna-chang-shopify !

**Pull request merged**